### PR TITLE
{storage-preview} Add SMB Multichannel support

### PR DESCRIPTION
**Pull request by Juliehzl**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
storage-preview

File: src/storage-preview/README.md [hunk 1]
```diff
@@ -262,4 +262,51 @@ az storage account network-rule remove \
...
+az storage account file-service-properties update \
+    --enable-delete-retention false \
+    -n mystorageaccount \
+    -g MyResourceGroup
+```
+
+##### Prepare for SMB multichannel
+###### Install private package
+
+`az extension add -s https://zuhdefault.blob.core.windows.net/cliext/storage_preview-0.6.1-py2.py3-none-any.whl`
```
Comment (evelyn-ys) [inline src/storage-preview/README.md:286]: Should this be a public package after release?

Comment (Juliehzl) [reply src/storage-preview/README.md:286]: good point

Comment (Juliehzl) [reply src/storage-preview/README.md]: updated

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**